src/if_ruby.c

```diff
@@ -982,6 +1021,19 @@ static void error_print(int state)
 	    if (p) *p = '\0';
 	    EMSG(buff);
 	}
+
+	attr = syn_name2attr((char_u *)"Error");
+# ifdef RUBY21_OR_LATER
+	bt = rb_funcallv(error, rb_intern("backtrace"), 0, 0);
+	for (i = 0; i < RARRAY_LEN(bt); i++) {
```

Comment:
The `{` should be on the next line.
See `:h style-examples`.

Reply:
Thank you!
Fixed 822ce02